include/sdf/SDFImpl.hh

```diff
@@ -182,6 +253,13 @@ namespace sdf
     /// \return a wrapped clone of the SDF element
     public: static ElementPtr WrapInRoot(const ElementPtr &_sdf);
 
+    /// \brief wraps the SDF element into a root element with the version info.
+    /// \param[in] _sdf the sdf element. Will be cloned by this function.
+    /// \param[out] _errors Vector of errors.
+    /// \return a wrapped clone of the SDF element
+    public: static ElementPtr WrapInRoot(const ElementPtr &_sdf,
+                                         sdf::Errors &_errors);
```

Comment:
`sdf::Errors` should be the first argument to be consistent with the rest of the API.

Reply:
https://github.com/gazebosim/sdformat/pull/1295/commits/bdac90ca501ac9f8b455c66d9e37dd588466e8a5